caringcaribou/utils/iso14229_1.py

```diff
@@ -293,19 +295,26 @@ def receive_response(self, wait_window):
         return response
 
     @staticmethod
-    def is_positive_response(response):
+    def is_positive_response(response: List[int]) -> bool:
         """
         Returns a bool indicating whether 'response' is positive
 
         :param response: ISO-14229-1 response data
-        :return: False if response is a NEGATIVE_RESPONSE,
-                 True otherwise
-        """
-        if (response is not None and
-           len(response) > 0 and
-           response[0] != Constants.NR_SI):
-            return True
-        return False
+        :return: False if response is a NEGATIVE_RESPONSE, True otherwise
```

Comment:
I believe this description would benefit from being a bit more specific. The current wording misses the case where the response has length 0, as this function will return False even though the response is *not* a NEGATIVE_RESPONSE.

So perhaps something along the lines of `False if response is invalid or a NEGATIVE_RESPONSE, True otherwise`

Reply:
Thats a good point 👍

Reply:
What do you think about addint here handling also None as a positive response for requests with set Suppress Positive Response Bit?